server.go

```diff
@@ -332,6 +333,34 @@ func listenToServers() error {
 /*
 Implementation of different RPC methods exported by server to clients
 */
+type ServerClient int
+
+func (t *ServerClient) ServerPut(key string, value string, clientId int, serverId int, clientClock int, reply *shared.Clock) error {
```

Comment:
I don't think you can have arbitrarily more number of arguments. The RPC methods should take exactly two arguments: one for requestArgs and one for response (which should be a pointer). so, combine the first five fields into a struct named ServerPutArgs under shared/shared_structs.go and call from the client code accordingly.